travis.sh

```diff
@@ -233,8 +234,14 @@ source /opt/ros/$ROS_DISTRO/setup.bash # re-source setup.bash for setting enviro
 # for catkin
 if [ "${TARGET_PKGS// }" == "" ]; then export TARGET_PKGS=`catkin_topological_order ${CI_SOURCE_PATH} --only-names`; fi
 if [ "${PKGS_DOWNSTREAM// }" == "" ]; then export PKGS_DOWNSTREAM=$( [ "${BUILD_PKGS// }" == "" ] && echo "$TARGET_PKGS" || echo "$BUILD_PKGS"); fi
-if [ "$BUILDER" == catkin ]; then catkin build -i -v --summarize  --no-status $BUILD_PKGS $CATKIN_PARALLEL_JOBS --make-args $ROS_PARALLEL_JOBS            ; fi
-
+if [ "$BUILDER" == catkin ]; then
+    _COMMAND_CATKINBUILD="catkin build -i -v --summarize  --no-status $BUILD_PKGS $CATKIN_PARALLEL_JOBS --make-args $ROS_PARALLEL_JOBS"
```

Comment:
why do you use ``-v`` for your catkin commands?
 
>  --verbose, -v         Print output from commands in ordered blocks once the
>                        command finishes.

I don't use that flag locally in my development but I'm able to see enough feedback from catkin to fix bugs when they occur. I think that would save a lot of log space (and moveit doesn't have any to spare)